app/js/app/directives/react_classes/ContentBlock.js

```diff
@@ -37,8 +47,21 @@ define(["react", "jquery"], function(React,$) {
 					var children = <ContentValueOrChildren value={this.props.doc.value} children={this.props.doc.children} disableListOps={this.props.disableListOps} encoding={this.props.doc.encoding} onChange={this.onContentChange}/>;
 				}
 
+				if (this.props.doc.type === "isaacQuiz") {
+					var rubric = <div>
+						<h3>Rubric</h3>
+						<ContentValueOrChildren blockTypeTitle="rubric" value={this.props.doc.rubric.value} children={this.props.doc.rubric.children} onChange={this.onRubricChange}/>
+					</div>;
+
+					var quizSectionHeader = <div>
+						<h3>Quiz Sections</h3>
+					</div>
+				}
+
 				return (
 					<Block type="content" blockTypeTitle={this.props.blockTypeTitle} doc={this.props.doc} onChange={this.onDocChange}>
+						{rubric}
```

Comment:
Is this safe when `rubric` is undefined on any other page type or content block? I assume you'd have noticed if it wasn't?

Reply:
Yeah, I'll have a double check when I look at the wording again but we do this elsewhere with 'conditional' sections, there's just a long section of bits in {} and they don't render if they're undefined